### PR TITLE
Ac 127201331 edit referrals

### PR DESCRIPTION
**Pull request by nate-cc**
not finished yet just thought i would float what i have done so far by you suman

### CODE REVIEW

**Comment by cc-integrations** [comment]
Why is the test failing?

File: bin/docker_start [hunk 1]
```diff
@@ -4,4 +4,4 @@ bundle check || bundle install
 npm install
 bin/gulp
 
-bin/gulp server & bundle exec puma
+bin/gulp && while true; do bin/gulp watch; done & bundle exec puma
```
Comment (cc-integrations) [inline bin/docker_start]: can we have this set up in Gulpfile.js

gulp.task('watch', ['server'], function() {
   gulp.watch(['app/assets/javascripts/**/*'], ['build-and-version-assets'])
 })

here when we run task server, then it watches for javascript changes

Comment (nate-cc) [inline bin/docker_start]: @pemacc LOG OUT OF THE SHARED ACCOUNT

**Comment by nate-cc** [comment]
can you please review asap. Need to progress for interview today

**Pull request closed without merging**